installing.md

```diff
@@ -0,0 +1,288 @@
+
+`production`: contains only code that has been tested against a full-planet build and is live on
+Mapzen Search. This is the "safest" branch and it will change the least frequently, although we
+generally release new code at least once a week.
+
+`staging`: these branches contain the code that is currently being tested against a full planet
+build for imminent release to Mapzen Search. It's useful to track what code will be going out in the
+next release, but not much else.
+
+`master`: master branches contain the latest code that has passed code review, unit/integration
+tests, and is ready to be included in the next release. While we try to avoid it, the nature of the
+master branch is that it will sometimes be broken. That said, these are the branches to use for
+development of new features.
+
+## Choose your datasets
+
+Pelias can currently import data from four different sources. The contents and description of these
+sources are available on our [data sources page](./data_sources). Here we'll just focus on what to
+download for each one.
+
+### Whosonfirst
+
+There are two ways to download Whosonfirst data. The first is to use the pre-created
+[bundles](https://whosonfirst.mapzen.com/bundles/). These consist of a series of archives that can
+be easily extracted (instructions are on the page).
+
+For more advanced uses, or to contribute back to Whosonfirst, use the
+[whosonfirst-data](https://github.com/whosonfirst/whosonfirst-data) Github repository. Again, there
+are [instructions](https://github.com/whosonfirst/whosonfirst-data#git-and-github). Note that this
+repo requires [git-lfs](https://git-lfs.github.com/), a lot of bandwidth, and 27GB (currently) of
+disk space.
+
+### Geonames
+
+The [pelias/geonames](https://github.com/pelias/geonames/#importing-data) importer contains code and
+instructions for downloading Geonames data automatically. Individual countries, or the entire planet
+(1.3GB) can be specified.
+
+### Openaddresses
+The Openaddresses project includes [numerous download options](https://results.openaddresses.io/),
+all of which are `.zip` downloads. The full dataset is several gigabytes, but there are numerous
+subdivision options. In any case, the `.zip` files simply need to be extracted to a directory of
+your choice, and Pelias can be configured to either import every `.csv` in that directory, or only
+selected files.
+
+### Openstreetmap
+
+Openstreetmap has a nearly limitless array of download options, and any of them should work as long as
+they're in [PBF](http://wiki.openstreetmap.org/wiki/PBF_Format) format. Generally the files will
+have the extension `.osm.pbf`. Good sources include the [Mapzen Metro Extracts](https://mapzen.com/data/metro-extracts/)
+(feel free to submit pull requests for additional cities or regions if needed), and planet files
+listed on the [OSM wiki](http://wiki.openstreetmap.org/wiki/Planet.osm).
+
+
```

Comment:
Let's make sure the capitalization is correct in cases where these are being used not as code, such as OpenStreetMap.